src/registrar/forms/domain_request_wizard.py

```diff
@@ -78,6 +79,20 @@ def clean_sub_organization(self):
         # Otherwise just return the suborg as normal
         return self.cleaned_data.get("sub_organization")
 
+    def clean_requested_suborganization(self):
+        name = self.cleaned_data.get("requested_suborganization")
+        if (
+            name
+            and Suborganization.objects.filter(
+                name__iexact=name, portfolio=self.domain_request.portfolio, name__isnull=False, portfolio__isnull=False
+            ).exists()
+        ):
+            raise ValidationError(
+                "This suborganization already exists. "
```

Comment:
I like the error message that appears at the top of the page when a suborganization already exists; however, we should show the same error at the field level, as well. Highlight the "requested suborganization" field in red  and display the error: This suborganization already exists.